src/bloqade/pyqrack/squin/qubit.py

```diff
@@ -71,15 +71,23 @@ def measure_and_reset(
         result = []
         for qbit in qubits:
             if qbit.is_active():
-                result.append(qbit.sim_reg.m(qbit.addr))
+                qbit_result = bool(qbit.sim_reg.m(qbit.addr))
             else:
-                result.append(None)
-            qbit.sim_reg.force_m(qbit.addr, 0)
+                qbit_result = None
```

Comment:
The Interpreter has a value `interp.loss_m_result` which stores the value of the measurement of a lost qubit.

Reply:
I think we should update the `Measure` statement to return this enum instead of a bool actually this would be consistent with my discussions with @cduck

Reply:
I think we should split that into another PR though